src/bin/main.rs

```diff
@@ -0,0 +1,127 @@
+extern crate observer;
```

Comment:
Lets create an examples folder, and put example code there.

Reply:
Ok